rules/opentitan/cc.bzl

```diff
@@ -477,3 +479,83 @@ opentitan_binary_assemble = rule(
     },
     toolchains = [LOCALTOOLS_TOOLCHAIN],
 )
+
+def _obj_transform_impl(ctx):
```

Comment:
I'd like to see these functions moved to something like //rules/opentitan:legacy.bzl, as they each have exactly one use in the codebase and some future work should evaluate whether or not they can simply be eliminated.